adhocracy4/comments/static/comments/Comment.jsx

```diff
@@ -8,12 +8,15 @@ var CommentManageDropdown = require('./CommentManageDropdown')
 
 var React = require('react')
 var django = require('django')
-var moment = require('moment')
 
 var safeHtml = function (text) {
   return { __html: text }
 }
 
+var localeDate = function (dateStr) {
+  return new Date(dateStr).toLocaleString(document.documentElement.lang)
```

Comment:
This values comes from our api, which by [default](http://www.django-rest-framework.org/api-guide/fields/#datefield) uses is8601. According to [mdn](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse) this is fine since ecma script 5.